grunt/helpers.js

```diff
@@ -196,11 +196,30 @@ module.exports = function(grunt) {
         var includes = grunt.config('includes');
         var excludes = grunt.config('excludes');
 
+        var isIncluded;
+        var isExcluded;
+
         // carry on as normal if no includes/excludes
         if (!includes && !excludes) return true;
 
-        var isIncluded = includes && pluginPath.search(exports.getIncludedRegExp()) !== -1;
-        var isExcluded = excludes && pluginPath.search(exports.getExcludedRegExp()) !== -1;
+        // get plugins directory within theme/less
+        var compareVar = grunt.file.doesPathContain((grunt.config('sourcedir') + 'theme' + path.sep + grunt.config('theme') + path.sep + 'less' + path.sep + 'src' + path.sep + 'plugins'), pluginPath);
```

Comment:
Can you use `path.join()` here instead of all the concatenation?

Also, `compareVar` would be better called `isThemePluginsFolder` for readability

Reply:
Thanks @brian-learningpool will make the `path.join()` change.

Could you maybe voice your opinion on the naming of the variable in the other comment please?